modules/rest-api/pages/rest-configure-memory.adoc

```diff
@@ -16,12 +16,17 @@ POST /pools/default
 [#description]
 == Description
 
-Allows a custom memory quota to be established for the Data, Index, Search, Eventing, and Analytics Services.
-(The Query and Backup Services do not require a memory allocation.)
-If no custom quota is specified for one or more services, those services retain the default allocations: these are, for the Data Service 256 Mb; for the Search Service 256 Mb; for the Index Service 512 Mb; for the Eventing Service 256 Mb; and for the Analytics Service 1024 Mb.
-Note that in each case except the Analytics Service, the minimum allowed allocation is 256 Mb: for the Analytics Service, the minimum allowed allocation is 1024 Mb.
+Allows a custom memory quota to be established for the Multi-Dimensional Scaling (MDS) services Index, Search, Eventing, and Analytics.
```

Comment:
Data needs to be in there -- this is a REST API command for configuring all services memory quota.

Allows a custom memory quota to be established for the Multi-Dimensional Scaling (MDS) services Data, Index, Search, Eventing, and Analytics.